res/res_pjsip_endpoint_identifier_ip.c

```diff
@@ -187,6 +187,31 @@ static int header_identify_match_check(void *obj, void *arg, int flags)
 		return 0;
 	}
 
+	/* Match match_header against request uri as well */
+	if (!strcasecmp(identify->match_header_name, "request")) {
+		int req_len;
+		char req_buf[PATH_MAX];
+		/* Print request line to req_buf */
+		req_len = pjsip_uri_print(PJSIP_URI_IN_REQ_URI, rdata->msg_info.msg->line.req.uri, req_buf, sizeof(req_buf) - 1);
```

Comment:
This new behavior doesn't appear to be documented anywhere, as well this is not part of the headers so I don't like reusing the header matching option. I think a new match option should be added instead, and its behavior documented as well.

Reply:
I've created the "request" identifier type, added documentation, migration, sample config, etc. Please note, this one will definitely conflict with https://github.com/asterisk/asterisk/pull/602  therefor I will rewrite one of the two after the the other is merged.